eZ/Bundle/EzPublishCoreBundle/SiteAccess/Config/ComplexConfigProcessor.php

```diff
@@ -23,12 +24,18 @@ final class ComplexConfigProcessor
     /** @var \eZ\Publish\Core\MVC\Symfony\SiteAccess\SiteAccessService */
     private $siteAccessService;
 
+    /** @var \eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\ComplexSettings\ComplexSettingParserInterface */
+    private $complexSettingParser;
+
     public function __construct(
         ConfigResolverInterface $configResolver,
         SiteAccessService $siteAccessService
     ) {
         $this->configResolver = $configResolver;
         $this->siteAccessService = $siteAccessService;
+
+        // instantiate non-injectable DI configuration parser
+        $this->complexSettingParser = new ComplexSettingParser();
```

Comment:
IMHO not worth to make it a service and probably not entirely correct, since it's a part of DI extension config processors. Anyway, it's been like this before (see the line 42).